src/CommunityToolkit.Maui/Categories/LogLevels.shared.cs

```diff
@@ -0,0 +1,17 @@
+namespace CommunityToolkit.Maui.Categories;
+
+/// <summary>
+/// Level of detail of the debug/logging messages.
+/// </summary>
+public enum LogLevels
```

Comment:
Well, it depends. 

I just implement two log levels, so if we use the "native" .net loglevel class, people can expect warnings, errors, and etc. It is strictly for debugging purposes, in case a developer needs.

Reply:
I think we should use standard .net logger if we want to add logging

Reply:
Ok, up to you. ✌️